### PR TITLE
small doc fix

### PR DESCRIPTION
**Pull request by KevinMenden**
## PR checklist
 - fixed a small detail in the docs - that's it

### CODE REVIEW

**Comment by lpantano** [comment]
@KevinMenden can you do the PR to the devel branch no the master?

**Comment by drpatelh** [comment]
Changed the base branch to `dev` but you might have to push a commit to re-trigger the tests.

**Comment by KevinMenden** [comment]
sorry didn't think of that ...

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/nfcore/smrnaseq.svg)](https://hub.docker.com/r/nfcore/smrnaseq)
 
+[![DOI](https://zenodo.org/badge/140590861.svg)](https://zenodo.org/badge/latestdoi/140590861)
+
+
```
Comment (lpantano) [inline README.md]: maybe remove these two empty lines and that way we can trigger the travis test again?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/nfcore/smrnaseq.svg)](https://hub.docker.com/r/nfcore/smrnaseq)
 
+[![DOI](https://zenodo.org/badge/140590861.svg)](https://zenodo.org/badge/latestdoi/140590861)
+
```
Comment (ewels) [inline README.md]:
```suggestion

```

**Comment by ewels** [comment]
I tried doing the commit via a suggestion but I don't have permission, which probably means the "allow contributors to commit to this PR" check box was left unticked.

Anyway, it's just markdown so I will admin override the tests and merge 👍

**Comment by ewels** [comment]
Can keep an eye on the merge build, which _should_ pass: https://travis-ci.com/nf-core/smrnaseq/builds/147896834

**Pull request merged**